docs/markdown/expert/hooks/02-hooks/03-useEffect.md

```diff
@@ -0,0 +1,34 @@
+<!-- .slide: class=""with-code““ -->
```

Comment:
this slide does not fit the screen for me, I am on a MacBook Pro mid-2015 and sadly it's not a 1980x1080 but a 1440x900
<img width="1355" alt="Capture d’écran 2021-04-20 à 12 45 35" src="https://user-images.githubusercontent.com/24412934/115383691-743a8e80-a1d6-11eb-96fe-fbe479b4abc2.png">

Reply:
This slide has clearly to much text IMO.  
I will split it in two slides. Thanks :)